docs/README.md

```diff
@@ -12,6 +12,7 @@
 - [Notifications](io.cozy.notifications.md): notifications made by the apps
 - [Permissions](io.cozy.permissions.md): Permissions of the instance
 - [Photos Albums](io.cozy.photos.albums.md): photos albums
+- [Procedures](io.cozy.procedures.md): Administrative procedures
```

Comment:
Wouldn't it be better to name it io.cozy.administrativeprocedure ? Simply "procedure" seems overly generic

Reply:
:+1: